### ISSUE TITLE
Manual test run on Linux for 0.19.x Hotfix 2 (Release channel)

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#11631]
## Per release specialty tests
 - [x] Upgrade to muon 4.5.x. ([#11498](https://github.com/brave/browser-laptop/issues/11498))
 - [x] Upgrade to Chromium 62. ([#11139](https://github.com/brave/browser-laptop/issues/11139))

## Installer

- [x] Check that installer is close to the size of last release.
- [ ] Check signature: If OS Run `spctl --assess --verbose /Applications/Brave.app/` and make sure it returns `accepted`.  If Windows right click on the installer exe and go to Properties, go to the Digital Signatures tab and double click on the signature.  Make sure it says "The digital signature is OK" in the popup window.
- [x] Check Brave, muon, and libchromiumcontent version in About and make sure it is EXACTLY as expected.

## Last changeset test

- [ ] Test what is covered by the last changeset (you can find this by clicking on the SHA in about:brave).

## Widevine/Netflix test

- [ ] Test that you can log into Netflix and start a show.

## Ledger
- [x] Verify wallet is auto created after enabling payments
- [x] Verify monthly budget and account balance shows correct BAT and USD value
- [ ] Verify the wallet overlay is shown when wallet transition is happening upon upgrade 
- [x] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
- [x] Verify that Brave BAT wallet address can be copied
- [ ] Verify adding funds via any of the currencies flows into BAT Wallet after specified amount of time
- [ ] Verify adding funds to an existing wallet with amount, adjusts the BAT value appropriately 
- [ ] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
- [x] Visit a verified publisher like brianbondy.com and ensure the verified publisher icon is shown on the ledger table
- [x] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table.
- [x] Check that disabling payments and enabling them again does not lose state.

## Sync
- [x] Verify you are able to sync two devices using the secret code
- [x] Visit a site on device 1 and change shield setting, ensure that the saved site preference is synced to device 2
- [x] Enable Browsing history sync on device 1, ensure the history is shown on device 2 
- [x] Import/Add bookmarks on device 1, ensure it is synced on device 2 
- [x] Ensure imported bookmark folder structure is maintained on device 2
- [x] Ensure bookmark favicons are shown after sync

## Data

- [ ] Make sure that data from the last version appears in the new version OK.
- [ ] Test that the previous version's cookies are preserved in the next version.

## About pages

- [x] Test that about:adblock loads
- [x] Test that about:autofill loads
- [x] Test that about:bookmarks loads bookmarks
- [x] Test that about:downloads loads downloads
- [x] Test that about:extensions loads
- [x] Test that about:history loads history
- [x] Test that about:passwords loads
- [x] Test that about:styles loads
- [x] Test that about:welcome loads
- [x] Test that about:preferences changing a preference takes effect right away
- [x] Test that about:preferences language change takes effect on re-start

## Bookmarks

- [x] Test that creating a bookmark on the bookmarks toolbar with the star button works
- [x] Test that creating a bookmark on the bookmarks toolbar by dragging the un/lock icon works
- [x] Test that creating a bookmark folder on the bookmarks toolbar works
- [x] Test that moving a bookmark into a folder by drag and drop on the bookmarks folder works
- [x] Test that clicking a bookmark in the toolbar loads the bookmark.
- [x] Test that clicking a bookmark in a bookmark toolbar folder loads the bookmark.

## Context menus
- [x] Make sure context menu items in the URL bar work
- [x] Make sure context menu items on content work with no selected text.
- [x] Make sure context menu items on content work with selected text.
- [x] Make sure context menu items on content work inside an editable control on `about:styles` (input, textarea, or contenteditable).

## Find on page

- [x] Ensure search box is shown with shortcut
- [x] Test successful find
- [x] Test forward and backward find navigation
- [x] Test failed find shows 0 results
- [x] Test match case find

## Geolocation

- [x] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation works

## Site hacks

- [x] Test https://www.twitch.tv/adobe sub-page loads a video and you can play it

## Downloads

- [x] Test downloading a file works and that all actions on the download item works.

## Fullscreen

- [x] Test that entering full screen window works View -> Toggle Full Screen. And exit back (Not Esc).
- [x] Test that entering HTML5 full screen works. And Esc to go back. (youtube.com)

## Tabs, Pinning and Tear off tabs

- [x] Test that tabs are pinnable
- [x] Test that tabs are unpinnable
- [x] Test that tabs are draggable to same tabset
- [x] Test that tabs are draggable to alternate tabset
- [x] Test that tabs can be teared off into a new window
- [x] Test that you are able to reattach a tab that is teared off into a new window
- [x] Test that tab pages can be closed
- [x] Test that tab pages can be muted

## Zoom

- [x] Test zoom in / out shortcut works
- [x] Test hamburger menu zooms.
- [x] Test zoom saved when you close the browser and restore on a single site.
- [x] Test zoom saved when you navigate within a single origin site.
- [x] Test that navigating to a different origin resets the zoom

## Bravery settings

- [x] Check that HTTPS Everywhere works by loading https://https-everywhere.badssl.com/
- [x] Turning HTTPS Everywhere off and shields off both disable the redirect to https://https-everywhere.badssl.com/
- [x] Check that ad replacement works on http://slashdot.org
- [x] Check that toggling to blocking and allow ads works as expected.
- [x] Test that clicking through a cert error in https://badssl.com/ works.
- [x] Test that Safe Browsing works (http://downloadme.org/)
- [x] Turning Safe Browsing off and shields off both disable safe browsing for http://downloadme.org/.
- [x] Visit https://brianbondy.com/ and then turn on script blocking, nothing should load. Allow it from the script blocking UI in the URL bar and it should work.
- [x] Test that about:preferences default Bravery settings take effect on pages with no site settings.
- [x] Test that turning on fingerprinting protection in about:preferences shows 3 fingerprints blocked at https://jsfiddle.net/bkf50r8v/13/. Test that turning it off in the Bravery menu shows 0 fingerprints blocked.
- [x] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/9/ when 3rd party cookies are blocked and not blank when 3rd party cookies are unblocked.
- [x] Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ when fingerprinting protection is on.
- [x] Test that browser is not detected on https://extensions.inrialpes.fr/brave/

## Content tests

- [x] Go to https://brianbondy.com/ and click on the twitter icon on the top right. Test that context menus work in the new twitter tab.
- [x] Load twitter and click on a tweet so the popup div shows.   Click to dismiss and repeat with another div. Make sure it shows.
- [x] Go to https://www.bennish.net/web-notifications.html and test that clicking on 'Show' pops up a notification asking for permission. Make sure that clicking 'Deny' leads to no notifications being shown.
- [x] Go to https://trac.torproject.org/projects/tor/login and make sure that the password can be saved. Make sure the saved password shows up in `about:passwords`. Then reload https://trac.torproject.org/projects/tor/login and make sure the password is autofilled.
- [x] Open `about:styles` and type some misspellings on a textbox, make sure they are underlined.
- [x] Make sure that right clicking on a word with suggestions gives a suggestion and that clicking on the suggestion replaces the text.
- [x] Make sure that Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link opens a new tab but does NOT switch to it.  Click on it and make sure it is already loaded.
- [x] Open an email on http://mail.google.com/ or inbox.google.com and click on a link. Make sure it works.
- [x] Test that PDF is loaded at http://www.orimi.com/pdf-test.pdf
- [x] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).

## Flash tests
- [x] Turn on Flash in about:preferences#security. Test that clicking on 'Install Flash' banner on myspace.com shows a notification to allow Flash and that the banner disappears when 'Allow' is clicked.
- [x] Test that flash placeholder appears on http://www.homestarrunner.com

## Autofill tests
- [x] Test that autofill works on http://www.roboform.com/filling-test-all-fields

## Session storage

Do not forget to make a backup of your entire `~/Library/Application\ Support/Brave` folder.

- [x] Temporarily move away your `~/Library/Application\ Support/Brave/session-store-1` and test that clean session storage works. (`%appdata%\Brave in Windows`, `./config/brave` in Ubuntu)
- [x] Test that windows and tabs restore when closed, including active tab.
- [x] Move away your entire `~/Library/Application\ Support/Brave` folder (`%appdata%\Brave in Windows`, `./config/brave` in Ubuntu)

## Cookie and Cache

- [x] Make a backup of your profile, turn on all clearing in preferences and shut down.  Make sure when you bring the browser back up everything is gone that is specified.
- [x] Go to http://samy.pl/evercookie/ and set an evercookie. Check that going to prefs, clearing site data and cache, and going back to the Evercookie site does not remember the old evercookie value.

## Update tests

- [ ] Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.

**Comment by LaurenWags** [comment]
Verified Flash section by using '--no-sandbox' in terminal.

**Comment by srirambv** [comment]
Closing in favour of #11668
